session.go

```diff
@@ -12,13 +12,14 @@ type SessionResolverBuilder interface {
 
 type SessionResolver interface {
 	SetDataFromSessionToContext(c context.Context) (context.Context, error)
-	SetDataToContext(session Session, c context.Context) (context.Context, error)
```

Comment:
why did you change this ?

Reply:
renamed to SetDataFromCurrentSessionToContext. because we have SetDataFromSessionToContext. and they do the same

Reply:
We're in session "domain"
code calling this would look like this:
seesion.SetDataToContext
which makes more sense than 
seesion.SetDataFromCurrentSessionToContext

SO...perhaps the change is the other way around
SetDataFromSessionToContext => SetDataToContext